unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTShapeLoaderController.cs

```diff
@@ -148,6 +150,7 @@ IEnumerator FetchNFTImage()
                 OnLoadingAssetFail?.Invoke();
             });
 
+        yield return new WaitUntil(() => Vector3.Distance(CommonScriptableObjects.playerUnityPosition, transform.position) < 30f);
```

Comment:
```suggestion
        yield return new WaitUntil(() => (transform.position - CommonScriptableObjects.playerUnityPosition).sqrMagnitude < 900f );
```
Wouldn't be better (performance wise) to use squared distance here?

Reply:
good catch!

Reply:
Done!.